File: VB/WindowsFormsApplication2/Form1.vb
Imports DevExpress.XtraTreeList.Columns
Imports System
Imports System.Collections.Generic
Imports System.Data
Imports System.IO
Imports System.Linq
Imports System.Windows.Forms

Namespace WindowsFormsApplication2
	Partial Public Class Form1
		Inherits Form

		Public Sub New()
			InitializeComponent()
			SetDataSource()
		End Sub

		Private Sub SetDataSource()
			Dim [set] As New DataSet()
			[set].ReadXml(Directory.GetParent(Directory.GetCurrentDirectory()).Parent.FullName & "\Contacts.xml")
			treeList1.DataSource = [set].Tables(0)
			treeList1.Columns("Description").Visible = False
			treeList1.BestFitColumns()
		End Sub
	End Class
End Namespace

File: CS/WindowsFormsApplication2/Form1.cs
using DevExpress.XtraTreeList.Columns;
using System;
using System.Collections.Generic;
using System.Data;
using System.IO;
using System.Linq;
using System.Windows.Forms;

namespace WindowsFormsApplication2
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
            SetDataSource();
        }

        private void SetDataSource()
        {
            DataSet set = new DataSet();
            set.ReadXml(Directory.GetParent(Directory.GetCurrentDirectory()).Parent.FullName + @"\Contacts.xml");
            treeList1.DataSource = set.Tables[0];
            treeList1.Columns["Description"].Visible = false;
            treeList1.BestFitColumns();
        }
    }
}
